easybuild/easyconfigs/c/ColabFold/ColabFold-1.5.5-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -175,11 +175,4 @@ modloadmsg = """
 %(name)s notebooks are located in $EBROOTCOLABFOLD/notebooks
 """
 
-postinstallmsgs = [
```

Comment:
@ThomasHoffmann77 apologies for this extra hassle, but I thought you were going to keep this text and put it as a comment? Seems like valuable information.

Reply:
Thought the comments in line 169 were enough. However, I'll add some more infos tomorrow